### DISCUSSION TITLE
[non-GATool] Event Music

### DISCUSSION THREAD

**Discussion by timtim17** [arthurlockman/gatool-ui#293] [General]
Hi James,

We chatted briefly on Blair's call about music on event streams. Following up on the GATool GitHub as mentioned in the call chat.

In PNW, we have a soundboard that is able to output two separate mixes -- we have one "main" mix that goes to the venue, with copyrighted music (played off of Spotify). We have a second "stream" mix that is separate with its own music, that goes only to the streaming hardware. It could be possible to play the same music in both the venue and stream without that capability, but to avoid running afoul of copyright on Twitch the event should only run the non-copyright music, which would not include many fan favorites.

On the stream mix in PNW in particular, we utilize [Monstercat](https://www.monstercat.com/gold), a record label carrying generally electronic music. They offer a subscription for 'content creators' at under $10/month. PNW generally sets that and leaves it running on the stream for the entire event.

Tyler Olds has written from time to time about a similar solution for events broadcast on FUN. Seems he has a playlist that's a mix of Monstercat and another music library, [Epidemic Sound](https://www.epidemicsound.com/). https://www.chiefdelphi.com/t/lets-solve-the-twitch-and-youtube-dmca-issues-from-streams/429168/27

In Washington state for FTC, we have a volunteer that has compiled a playlist of songs on Spotify that have not had issues with copyright. This would be free (minus having a Spotify subscription), and we haven't had issues on our FTC broadcasts. Mostly from the label ["No Copyright Sounds"](https://ncs.io/) https://open.spotify.com/playlist/2dusAGlP49n9FNnFLmKvWr?si=8bc43655915c41bb

**Comment by jameslockman** [comment]
Thanks so much!
